### ISSUE TITLE
Extension of keyword Drag And Drop By Coordinates: make release of the mouse button flexible

### ISSUE THREAD

**Issue by UliSei** [MarketSquare/robotframework-browser#1716]
So far the keyword "Drag And Drop By Coordinates" does the following:

```
First it moves the mouse to the start-point,
then presses the left mouse button,
then moves to the end-point in specified number of steps,
then releases the mouse button.
```

I'd suggest to add a boolean parameter "release" which controls whether the mouse button at the end is released or not.

The default value would be True so that nothing would change for existing calls of the keyword.

The extended keyword used in sequence allows to drag the mouse several times to different coordinates before the mouse button is released.

This would allow a more realistic simulation of user interaction.

And it would allow to test GUI elements that react permanently on the position of the mouse like virtual joysticks.

Suggestion to Browser/keywords/interaction.py, not tested yet:

OLD:
```
def drag_and_drop_by_coordinates(
        self, from_x: float, from_y: float, to_x: float, to_y: float, steps: int = 1
    ):  ...
        self.mouse_button(MouseButtonAction.down, x=from_x, y=from_y)
        self.mouse_move(x=to_x, y=to_y, steps=steps)
        self.mouse_button(MouseButtonAction.up)
```

NEW:
```
def drag_and_drop_by_coordinates(
        self, from_x: float, from_y: float, to_x: float, to_y: float, steps: int = 1, release: bool = True
    ):  ...
        self.mouse_button(MouseButtonAction.down, x=from_x, y=from_y)
        self.mouse_move(x=to_x, y=to_y, steps=steps)
        if release:
            self.mouse_button(MouseButtonAction.up)
```

**Comment by aaltat** [comment]
I do not see problems with such change.

**Comment by UliSei** [comment]
Fine :-)
My first contributon was just about docu text, so I did not do any functional test with my forked version. This time I would have to test the change with a web application which is available in my docker environment. When I fork the library in GitHub and clone and change it locally - how could I test it? How do I have to handle the node dependencies? 
I offer to do the pull request but I would need some support.

**Comment by aaltat** [comment]
You need to manually install Python and NodeJS, but rest of the development details are handled with scripts and invoke. Please see https://github.com/MarketSquare/robotframework-browser/blob/main/CONTRIBUTING.md#development-environment

We also have a inbuilt test application which provides different range of static and dynamic pages. Running tests is explained in here https://github.com/MarketSquare/robotframework-browser/blob/main/CONTRIBUTING.md#testing

If you need to create new pages to test this enhancement, static pages are in here https://github.com/MarketSquare/robotframework-browser/tree/main/node/dynamic-test-app/static Dynamic test page is in here https://github.com/MarketSquare/robotframework-browser/tree/main/node/dynamic-test-app/src

Most likely I did forget to explain something, because I am blind for the detail, so please don't hesitate to ask if something is unclear.

**Comment by UliSei** [comment]
Thank you very much, I'll try.

**Comment by UliSei** [comment]
Agree. 
Thanks for that hint, just in time.
If there aren't any objections I am going to implement it next week with `no_drop` argument.

**Comment by UliSei** [comment]
I had to think about it last night and I'd suggest the argument
`drop` which defaults to _True_.
The advantage is a more intuitive usage which avoids double negation.

```
def drag_and_drop_by_coordinates(
        self, from_x: float, from_y: float, to_x: float, to_y: float, steps: int = 1, drop: bool = True
    ):  ...
        self.mouse_button(MouseButtonAction.down, x=from_x, y=from_y)
        self.mouse_move(x=to_x, y=to_y, steps=steps)
        if drop:
            self.mouse_button(MouseButtonAction.up)
```

**Comment by UliSei** [comment]
Results so far with the modified keyword "Drag And Drop By Coordinates":
1a) The new argument "drop" seems not to disturb
the existing test has passed:
![grafik](https://user-images.githubusercontent.com/89480399/155123392-87088aa1-3a35-4a9e-b4d3-6441f4f7c56d.png)


1b) However there are some test cases that failed.
I doubt whether it's because of "Drag And Drop By Coordinates".
Eventually it's specific to my Windows testing environment?
![grafik](https://user-images.githubusercontent.com/89480399/155123564-691fcfaf-0782-419c-b701-82a468e9ecfc.png)


2) Does drop=False fulfill the expectations?
a) I created a test script with an import line
`Library    Browser`
However the lib is not found:
`[ ERROR ] Error in file 'C:\Users\Uli-Laptop\projects\robotframework-browser\atest\test\UliSeiMoveJoystickCartesianDD.robot' on line 18: Importing library 'Browser' failed: ModuleNotFoundError: No module named 'Browser'`

It's the same when I use
`Library    ../../Browser`
`[ ERROR ] Error in file 'C:\Users\Uli-Laptop\projects\robotframework-browser\atest\test\UliSeiMoveJoystickCartesianDD.robot' on line 18: Importing library '../../Browser' failed: ModuleNotFoundError: No module named '.'`

b) I tried to learn how the Browser lib is referenced in other robot files.
The most astonishing to me is that by
`inv atest`
the Test Suite _Presenter Mode_ is run successfully.
And when I run
`robot presenter_mode.robot`
it fails:
`Error in file 'C:\Users\Uli-Laptop\projects\robotframework-browser\atest\test\06_Examples\presenter_mode.robot' on line 3: Importing library 'Browser' failed: ModuleNotFoundError: No module named 'Browser'`
In consequence the keywords are not found.
```
TEST Filling the text
Full Name: 	Presenter Mode.Filling the text
Start / End / Elapsed: 	20220222 11:23:51.092 / 20220222 11:23:51.092 / 00:00:00.000
Status: 	FAIL
Message: 	No keyword with name 'open browser' found.
```

c) I tested successfully that running robot files without imported Browser lib works.
I'd feel better to have the new feature tested before starting the pull request.
